scripts/transform-year-organizations.ts

```
@@ -273,7 +306,7 @@ async function main() {
 
       // Merge technologies (union, preserving existing)
       const techSet = new Set([...(existing.technologies || [])]);
-      (raw.tech_tags || []).forEach((t: string) => techSet.add(t));
+      (raw.tech_tags || []).forEach((t: string) => techSet.add(getStandardTechName(t)));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)